lib/stripe/api_operations/save.rb

```diff
@@ -50,6 +50,9 @@ def update(id, params = {}, opts = {})
       #   idempotency_key to be passed in the request headers, or for the
       #   api_key to be overwritten. See
       #   {APIOperations::Request.execute_resource_request}.
+      #
+      # The `save` method is DEPRECATED and will be removed in a future major
+      # version of the library. Use the `update` method on the resource instead.
```

Comment:
Should we move this up to the beginning right after `Creates or updates an API resource.` so that it does not get buried in this wall of text?

Reply:
Good call! I moved it to the top before `Creates or updates an API resource` because I think the deprecation note is more obvious that way. To me, it would feel more buried if it's between 2 lines describing what the method does.